src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
Yeah, although it would be nice for it to enforce constness, at the very least it acts as documentation to other developers.